### ISSUE TITLE
Accessibility - Users can't change editor font size

### ISSUE THREAD

**Issue by Ben8t** [kestra-io/kestra#1218]
### Feature description

An accessibility issue, in the editor users should be able to change the font size and font type

**Comment by Skraye** [comment]
Add custom action to this : 

![Image](https://github.com/kestra-io/kestra/assets/37600690/8cfe172a-f348-4192-993d-f09cd5a20db4)
